src/plugins/pluginUtils.ts

```diff
@@ -19,12 +19,12 @@ export function getMemberLevel(
   member: GuildMember | PartialGuildMember,
   guild: Guild
 ): number {
-  if (guild.ownerID === member.id) {
+  if (guild.ownerId === member.id) {
     return 99999;
   }
 
   for (const [id, level] of Object.entries(levels)) {
-    if (member.id === id || (member.roles && member.roles?.cache.has(id))) {
+    if (member.id === id || member.roles?.cache?.has(id as Snowflake)) {
```

Comment:
The PermissionLevels should be updated so the id is a Snowflake instead

Reply:
That is sadly not possible since ID is the index, and those must be either string or number

Reply:
> That is sadly not possible since ID is the index, and those must be either string or number

But `Snowflake` is of type string